### PR TITLE
Grammar and spelling corrections

### PR DESCRIPTION
**Pull request by timtyrrell**

### CODE REVIEW

File: 12_src/src/application.jsx [hunk 1]
```diff
@@ -3,10 +3,10 @@
 // Now is the time to meet the first binding that redux-react (https://github.com/gaearon/react-redux)
 // brings to us: the Provider component.
 
-// Provider is a React Component designed to be used as a wrapper of your application's root component. Its
-// purpose is to provide your redux instance to all your application's components. How it does that does not
-// really matters to us but just to let you know, it's using React's context feature (it's undocumented so you
-// don't have to know about it but if you're curious:
+// Provider is a React Component designed to be used as a wrapper of your application's root component. It's
```
Comment (happypoulp) [inline 12_src/src/application.jsx]: Are you sure about this one? I really meant to use the possessive form, but maybe the sentence formulation is not clear... I meant "The purpose of Provider is to provide...".

File: 12_src/src/home.jsx [hunk 2]
```diff
@@ -20,13 +20,13 @@
...
 import { connect } from 'react-redux'
-// We use the same ES6 import trick to get all action creators and produce a hash than we did with
+// We use the same ES6 import trick to get all action creators and produce a hash like we did with
 // our reducers. If you haven't yet, go get a look at our action creator (./actions-creator.js).
 import * as actionCreators from './action-creators'
 
-// The "connect" decorator takes as only parameter, a function that will select which slice of your
-// state you want to expose to you component. This function is logically called a "selector" and 
-// receive 2 parameters: the state of your store and the current props of your component.
+// The "connect" decorator takes as it's only parameter, a function that will select which slice of your
```
Comment (happypoulp) [inline 12_src/src/home.jsx]: I would be very to use the possessive form (its) here. Am I incorrect?

Comment (timtyrrell) [inline 12_src/src/home.jsx]: I may have been overzealous in my changes, you are probably right. ;)

File: 12_src/src/application.jsx (hunk 1 above)
Comment (timtyrrell) [inline 12_src/src/application.jsx]: I will defer to you again and switch it back.

**Comment by timtyrrell** [comment]
I made the possessive changes then force pushed the branch. Let me know what you think, wonderful tutorial, btw.

**Comment by happypoulp** [comment]
Looks good to me, thanks a lot for the many corrections!
Glad you liked the tutorial ;)!

**Pull request merged**